packages/apps/src/app/trade/Form.ts

```diff
@@ -1140,11 +1142,11 @@ export class TradeForm extends BaseElement {
         <span>${this.error['pool'] || this.error['trade']}</span>
       </div>
       <uigc-button
-        ?disabled=${this.readonly || this.isDisabled()}
+        ?disabled=${isDisabledBtn}
         class="confirm"
         variant="primary"
         fullWidth
-        @click=${this.onCtaClick}>
+        @click=${isDisabledBtn ? undefined : this.onCtaClick}>
```

Comment:
Better solution is disabling uigc-button pointer events on disabled state .. See:
https://github.com/galacticcouncil/apps/commit/4cc8aa4c2d7a68a40ffb129730b9c8e1c5b5facf
I fixed it in different branch .. :)